### PR TITLE
Write a test for enum type using jest.

### PR DESCRIPTION
**Pull request by hwanseung**
Write a test for enum type using jest.

ISSUE=#99,#100

### CODE REVIEW

**Review by yjaeseok — APPROVED** [review]
lgtm

File: test/test_interface.idl [hunk 1]
```diff
@@ -32,4 +32,13 @@ interface TestInterface {
   short shortMethod(short number);
   double doubleMethod(double number);
   string stringMethod(string string);
+
+  // enum
+  void EnumMethod(TestEnum enumValue);
```
Comment (romandev) [inline test/test_interface.idl]: nit: Should be enumMethod(...) in IDL.
Also, I think better name is voidMethodTestEnumArg(...)

File: test/test_interface.cc [hunk 2]
```diff
@@ -64,3 +64,7 @@ double TestInterface::DoubleMethod(double number) {
 const std::string TestInterface::StringMethod(const std::string& string) {
   return string;
 }
+
+void TestInterface::EnumMethod(const std::string& string) {
+  last_call_info_ = "EnumMethod(" + string + ")";
```
Comment (romandev) [inline test/test_interface.cc]: The string should be IDL definition like other methods.
"void voidMethodTestEnumArg(TestEnum)";

**Review by romandev — COMMENTED** [review]
I left some minor comments!

File: test/test_interface.idl (hunk 1 above)
Comment (hwanseung) [reply test/test_interface.idl]: i fixed it.

File: test/test_interface.cc (hunk 2 above)
Comment (hwanseung) [reply test/test_interface.cc]: i fixed it.

**Review by romandev — APPROVED** [review]

**Pull request merged**